lib/datasource/index.ts

```diff
@@ -1,30 +1,8 @@
+import fs from 'fs';
 import { logger } from '../logger';
 import { addMetaData } from './metadata';
 import * as versioning from '../versioning';
 
-import * as cargo from './cargo';
-import * as cdnjs from './cdnjs';
-import * as dart from './dart';
-import * as docker from './docker';
-import * as hex from './hex';
-import * as github from './github';
-import * as gitlab from './gitlab';
-import * as gitTags from './git-tags';
```

Comment:
We now have a mismatch between `git-tags` (autoloaded) and `gitTags`, right? As well as gitSubmodules, gradleVersion, rubyVersion and terraformProvivder.

Reply:
Should be fixed now